lib/libc/minimal/include/sys/types.h

```diff
@@ -33,25 +33,7 @@ typedef __SIZE_TYPE__ ssize_t;
 #if !defined(__off_t_defined)
 #define __off_t_defined
 
-#if defined(__i386) || defined(__x86_64) || defined(__ARC64__)
-typedef long int off_t; /* "long" works for all of i386, X32 and true 64 bit */
-#elif defined(__ARM_ARCH)
-typedef int off_t;
-#elif defined(__arc__)
-typedef int off_t;
-#elif defined(__NIOS2__)
-typedef int off_t;
-#elif defined(__riscv)
-typedef int off_t;
-#elif defined(__XTENSA__)
-typedef int off_t;
-#elif defined(__sparc__)
-typedef int off_t;
-#elif defined(__mips)
-typedef int off_t;
-#else
-#error "The minimal libc library does not recognize the architecture!\n"
-#endif
+typedef long off_t;
```

Comment:
`typdef long int off_t`

Reply:
checkpatch doesn't like this (check previous runs)